apps/sync/test/unit/crypto/stringconversion_test.js

```diff
@@ -85,6 +99,22 @@ suite('utils', () => {
           throw(Error);
     });
   });
+  suite('byteArrayToUtf16String', () => {
+    test('converts an Uint8Array to a UTF-16 string', () => {
+      var array = new Uint8Array(3);
+      array[0] = 0xe2;
+      array[1] = 0x82;
+      array[2] = 0xac;
```

Comment:
The string "UTF-16" can be removed from the function name because the default string in JS is UTF-16.